ipn/ipnlocal/local.go

```diff
@@ -453,6 +453,24 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 	}
 }
 
+// exitNodeIP returns the exit node IP from prefs.
```

Comment:
, or the zero value if an exit node is not in use or the current exit node is no longer in the netmap.